lib/mutant/mutator/node/block.rb

```diff
@@ -36,7 +36,32 @@ def mutate_body
           return unless body
           emit(body)
           emit_body_mutations
+
+          mutate_body_receiver
+        end
+
+        # Mutate method send in body scope of `send`
+        #
+        # @return [undefined]
+        #
+        # @api private
+        def mutate_body_receiver
+          return unless n_send?(body)
+
+          body_meta = AST::Meta::Send.new(body)
```

Comment:
I find it interesting that you extracted `send_meta` into a separate method, yet `body_meta` is inline within this method, yet it has more call sides.

For consistency I would probably extract `body_meta`, and I'd imagine you'd probably want to memoize it given it's used in more than one place.